setup.md

```diff
@@ -46,8 +46,8 @@ inline error checking, and visual debugging features.
 
 To install and run Flutter, your development environment must meet these minimum requirements:
 
-* **Operating Systems**: Mac or Linux (64-bit). Windows support is [in progress](https://github.com/flutter/flutter/issues/138).
-* **Disk Space**: 700 MB on Mac, 600 MB on Linux (does not include disk space for Xcode or Android Studio).
+* **Operating Systems**: Mac or Linux (64-bit). For Windows instructions (beta), see [Flutter on Windows](https://github.com/flutter/flutter/wiki/Flutter-on-Windows).
```

Comment:
We should eventually merge these instructions in so that this one document covers all platforms, rather than having some second-class citizen platforms on the wiki.